contrib/test/integration/critest.yml

```diff
@@ -30,15 +30,15 @@
   # https://bugzilla.redhat.com/show_bug.cgi?id=1414236
   # https://access.redhat.com/solutions/2897781
 - name: run critest validation
-  shell: "critest -c --runtime-endpoint /var/run/crio/crio.sock --image-endpoint /var/run/crio/crio.sock -s 'should not allow privilege escalation when true' v"
+  shell: "critest --runtime-endpoint /var/run/crio/crio.sock --image-endpoint /var/run/crio/crio.sock --ginkgo.skip='should not allow privilege escalation when true' v"
   args:
     chdir: "{{ ansible_env.GOPATH }}/src/github.com/kubernetes-incubator/cri-o"
   async: 5400
   poll: 30
   when: ansible_distribution in ['RedHat', 'CentOS']
 
 - name: run critest benchmarks
-  shell: "critest -c --runtime-endpoint /var/run/crio/crio.sock --image-endpoint /var/run/crio/crio.sock b"
+  shell: "critest --runtime-endpoint /var/run/crio/crio.sock --image-endpoint /var/run/crio/crio.sock b"
```

Comment:
the failure on RHEL is because this line is running _all_ CRI validation like above :) you need to change that `b` at the end with `--benchmark` (either with two dashes or just one, `-benchmark`, `--benchmark`)

Reply:
Whoops, should have seen that. Thanks!